pkg/services/accesscontrol/resourcepermissions/store.go

```diff
@@ -657,11 +662,17 @@ func (s *store) getPermissions(sess *db.Session, resource, resourceID, resourceA
 	return result, nil
 }
 
-func (s *store) createPermissions(sess *db.Session, roleID int64, resource, resourceID, resourceAttribute string, actions map[string]struct{}) error {
+func (s *store) createPermissions(sess *db.Session, roleID int64, resource, resourceID, resourceAttribute string, actions map[string]struct{}, permission string) error {
 	if len(actions) == 0 {
 		return nil
 	}
-	permissions := make([]accesscontrol.Permission, 0, len(actions))
+	permissionsToBeStored := len(actions)
```

Comment:
is this really needed? since this is just an dynamic slice. double check @eleijonmarck \